api/mysagw/identity/tests/test_membership_views.py

```diff
@@ -0,0 +1,337 @@
+import datetime
+
+import pytest
+from django.urls import reverse
+from psycopg2._range import DateRange
```

Comment:
Import from a private module? Shouldn't the model field where you're using it be of type `django.contrib.postgres.fields.DateRangeField`? (Or maybe something from an "official" import)

Reply:
Ooops... Forgot to double check when letting PyCharm handle the import :see_no_evil: 

Fixed.

Reply:
Nice!